src/notification/email/email.service.ts

```diff
@@ -0,0 +1,36 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import * as sendgrid from '@sendgrid/mail';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class EmailService {
+  readonly #emailAddress: string;
+  constructor(private readonly configService: ConfigService) {
+    const sendgridApiKey = this.configService.get('SENDGRID_API_KEY');
```

Comment:
`getOrThrow` instead of `get` and it's gonna have the same effect as the two if checks below.

Reply:
@aarbanas Actually, ignore this comment. The below-suggested lines are already doing both.